packages/amplify-console-hosting/src/utils/amplify-console-utils.js

```diff
@@ -84,9 +85,12 @@ function waitJobToSucceed(job, amplifyClient) {
 }
 
 async function httpPutFile(filePath, url) {
+  const proxy = process.env.HTTP_PROXY || process.env.HTTPS_PROXY;
+  const proxyOption = proxy ? { agent: new ProxyAgent() } : {};
```

Comment:
can you copy comment from other places like this that explains why we're not consuming `proxy` ?